src/state/lobby.ts

```diff
@@ -282,6 +282,16 @@ export class LobbyState implements LobbyActionHandlers {
     if (reason) user.send(M.sRoomFlagsUpdateFailed({ reason }));
   }
 
+  cRoomModFlagsUpdate(payload: NT.ClientModFlagsUpdate, user: UserState) {
+    debug(`cRoomModFlagsUpdate from ${user.name} for room ${user.room()?.id} with payload: ${JSON.stringify(payload)}`)
```

Comment:
Is this leftover from testing?

Despite the function being called "debug", I kinda just use this for general logging because I never set up a proper logger 😅

Unless there's a reason it should stick around, let's drop this log line